continuous_integration/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+mkdir build
+cd build
+cmake -DKatydid_ENABLE_TESTING=TRUE ..
+make -j3 
+make install
+
+export PATH=$(pwd)/bin:${PATH}
```

Comment:
Why are all of these variables exported here?

Reply:
Maybe there is no need for these once the executables are built.
I could setup a separate setup.sh script if necessary...